.github/workflows/ios-build.yml

```diff
@@ -2,38 +2,78 @@ name: Test iOS build
 on:
   pull_request:
     paths:
+      - '.github/workflows/ios-build.yml'
+      - 'RNReanimated.podspec'
       - 'ios/**'
       - 'Common/**'
+      - 'Example/ios/**'
+      - 'FabricExample/ios/**'
   push:
     branches:
       - main
 
 jobs:
   build:
-    # runs-on: macos-latest // issue: https://github.com/actions/virtual-environments/issues/4060
-    runs-on: macos-11
+    runs-on: macos-12
     strategy:
       matrix:
         working-directory: [Example, FabricExample]
     concurrency:
-      group: ios-${{ github.ref }}
-      cancel-in-progress: true
+      group: ios-${{ matrix.working-directory }}-${{ github.ref }}
     steps:
-      - name: checkout
+      - name: Check out Git repository
         uses: actions/checkout@v2
-      - name: Use Node.js 14
-        uses: actions/setup-node@v2
+
+      - name: Restore Reanimated node_modules from cache
+        uses: actions/cache@v2
```

Comment:
Is there a reason why we use `cache@v2` instead of `cache@v3`?

Reply:
Totally not, I will change it to `cache@v3`. Good catch!